### PR TITLE
Remove words 'easy', 'simple', and 'intuitive' (Issue  #273)

### PR DESCRIPTION
**Pull request by klapaukh**
I have edited the text to remove references to the words:
 * Easy
 * Simple
 * Intuitive

as these words are discouraged in the instructor training. This is following on from Issue #273 which brought attention to this.

### CODE REVIEW

**Comment by klapaukh** [comment]
This is also a pull request for instructor training checkout

File: _episodes/08-making-packages-R.md [hunk 1]
```diff
@@ -215,5 +215,5 @@ kelvin_to_celsius(273.15)
 > ## Creating a Package for Distribution
 >
 > 1. Create some new functions for your tempConvert package to convert from Kelvin to Fahrenheit or from Celsius to Kelvin or Fahrenheit.
-> 2. Create a package for our `analyze` function so that it will be easy to load when more data arrives.
+> 2. Create a package for our `analyze` function so that it will be can be reloaded or shared when more data arrives.
```
Comment (ErinBecker) [inline _episodes/08-making-packages-R.md]: "so that it will be can be reloaded" --> "so that it can be reloaded"

**Comment by ErinBecker** [comment]
@klapaukh - Thanks for putting this together. These changes look great to me. There is just one case where there are two extraneous words in your changes. I've marked them in line. I'll recommend this to the Maintainers to merge.

File: _episodes/08-making-packages-R.md (hunk 1 above)
Comment (klapaukh) [reply _episodes/08-making-packages-R.md]: Woops! Added a commit to fix that!

**Comment by ErinBecker** [comment]
Thanks @klapaukh!

**Comment by katrinleinweber** [comment]
Thanks for preparing this. If @chendaniely doesn't have any additional items to discuss here, I'll merge this soon :-)

**Pull request merged**